src/modules.c

```diff
@@ -543,7 +543,7 @@ Function global_table[] = {
   /* 252 - 255 */
   (Function) egg_snprintf,
   (Function) egg_vsnprintf,
-  (Function) egg_memset,
+  (Function) 0,                   /* was egg_memset -- use memset() instead */
```

Comment:
Comment should rather point to egg_bzero?

Reply:
its done